app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,18 +11,111 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-filters-sidebar {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .background {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 200;
```

Comment:
some magic numbers here, could this be less magical? like 10?

Reply:
`z-index` is essentially global, and here we're adding an overlay that needs to have a higher index than every other item on the page (like the floating navigation bar for example). There's two elements here in the overlay that need to be stacked closely (one above the other) but I left some room in between them. 

`z-index` can basically take any number, but I generally like to leave a bit of a gap between them so that all the numbers don't need to be adjusted if someone wants to put an element "between" the existing values. Like if you have one layer at 10, and one at 11, and then want to put something between those layers, you have to change things in multiple places. If you have two layers at 10 and 20 for example, you could put something at 15 and it would sit between them without having to change the others.

I'll take another look at it now though :+1:

Reply:
So for the shop and for Darkswarm in general, we have a cluster of values around 1-10 for some lower elements, and a cluster of other values around 90-100 for some higher elements...

Reply:
great, thanks for having a look! I think we can leave it as is then.